src/modules/75-cf/pages/feature-flags/components/AllEnvironmentsFlagsListing.module.scss

```
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2021 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+.flagEnvironmentsContainer {
+  border-radius: 12px;
+  height: 85px;
+  &.prod {
+    background-color: var(--purple-50);
+  }
+  &.nonProd {
+    background-color: var(--teal-50);
+  }
+}
+
+.flagEnvironmentStatus {
+  border-radius: 8px;
```

some of this `px` could be referred to by a var
```suggestion
  border-radius: var(--spacing-small);
```